site/topics/more-lists/more-lists.rst

```diff
@@ -1,80 +1,82 @@
 *********************
-List & Pointer Trivia
+Aliases & List Trivia
```

Comment:
Directory name doesn't seem to correspond to the content. `more-lists`, moreover, doesn't seem to describe anything at all. Is there any reason this file isn't within `lists` ? (As in, the directory can contain more than a single lecture.)